tsan.mixin

```diff
@@ -0,0 +1,10 @@
+{
+    "build": {
+        "tsan-gcc": {
```

Comment:
Are you using this mixin with `clang`? If yes, the name of the mixin seems misleading and should be changed.